wiki/WORKFLOW.md

```diff
@@ -0,0 +1,61 @@
+# Workflow
+
+This document provides an overview of the collaboration principles that drive our project. It includes information relative to the contribution workflow, the structure of meeetings and the organization of sprints and epics (iterations).
+
+> No document is auto-sufficient or frozen over time. This is also the case with the workflow of our team, which we will update as time goes by.
+
+## Contributing
+
+We follow an approach inspired by the [Rapid Cycle Workflow](https://gitlab.com/softeng-heigvd/teaching-heigvd-pdg-2020/guidelines/-/blob/master/WORKFLOW.md). We adapt it to match our specific use-case:
+
+- we're using GitHub, in order to centralize all of our services (issues, project management, milestones, hosting, continuous integration);
+- we want to make sysetmatic pull request reviews; and
+- we'll be using an external time tracking system.
+
+### Rapid Cycle Workflow
+
+1. Based on what was agreed on during meetings, file an issue for a task you'd like to accomplish. When choosing a task, choose something that you think should take you **less than 4 hours** to achieve. It's better to aim for a task that's too small rather than a task that's too big !
+	+ Use the dedicated issue template to file that task.
+	+ Fill in our time tracking system the expected duration of this task.
+2. Use your personal fork, and create a branch dedicated to the issue. Once you have your first commit ready, immediatly create a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
+3. Whenever you commit your code, log the amount of time that you have spent on your task. Try to be accurate; it will help us estimate future tasks more precisely !
+	+ Follow [this article](https://chris.beams.io/posts/git-commit/) on how to write a good commit messages. The Git history of your contribution will also be looked at during the review.
+	+ Add a description to your commit message if relevant.
+	+ When doing some peer coding, [create a commit with multiple authors](https://docs.github.com/en/enterprise/2.13/user/articles/creating-a-commit-with-multiple-authors).
+	+ If you have a question on your work, write it in the feed of the draft pull request.
+4. Once you are with your contribution, mark your draft as ready. Make sure that merging the pull request will close the issue created in `1.` !
```

Comment:
"Once you are with" should be "Once you are done with"

Reply:
Fixed in dc7289b9a40f3b37524a4e3217179d3e684d804c.